doc/modules/ensemble.rst

```diff
@@ -838,7 +818,135 @@ accessed via the ``feature_importances_`` property::
 
  * :ref:`sphx_glr_auto_examples_ensemble_plot_gradient_boosting_regression.py`
 
- .. _voting_classifier:
+.. _histogram_based_gradient_boosting:
+
+Histogram-Based Gradient Boosting
+=================================
+
+Scikit-learn 0.21 introduces two new experimental implementations of
```

Comment:
If we say they're experimental here, we need to have a marker to make sure we remember to fix it when we move it out of being experimental.

Reply:
hard to do since we don't have a target version. Making an estimator not experimental won't be trivial (lots of files will be changed) so I'm pretty sure we won't miss the user guide